pdf/differential_text_test/config.json5

```diff
@@ -15,12 +15,12 @@
         type: 'regex-counter',
         version: '1',
         stdstar: {
-          '^Min (\\w+ to \\w+): (.*)': {
+          '^Min (.* to .*): (.*)': {
```

Comment:
Best practice would be to use `.*?` if you're going to use a `.` here, since that would be the non-greedy version.

Reply:
Good point. There's an argument for `[a-zA-Z0-9_\-]+` here, which would be equivalent (for ascii input) to `\w+`, but also matching `-`. But `.*?`/`.+?` should be the same amount of backtracking so might as well keep it short